src/components/filters/Filters.tsx

```diff
@@ -23,7 +23,7 @@ interface FiltersProps {
 
 export const Filters = ({ filters, onChange, onClear }: FiltersProps) => {
   return (
-    <aside className="flex flex-col justify-between border-b bg-gray-50 p-5 pb-10 sm:min-h-full sm:border-r sm:border-b-0">
+    <aside className="fixed flex h-full flex-col justify-between overflow-scroll border-b bg-gray-50 p-5 pb-10 sm:min-h-full sm:border-r sm:border-b-0">
```

Comment:
There is an issue with displaying the footer. U need to change this part of `className`

```suggestion
    <aside className="fixed flex h-full flex-col justify-between overflow-scroll border-b bg-gray-50 p-5 pb-24 sm:min-h-full sm:border-r sm:border-b-0">
```